<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微軟正黑體</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>OneClick Drivers Installer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="driversFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>461</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QGroupBox" name="landriGroupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>251</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>有線網絡介卡</string>
     </property>
     <widget class="QComboBox" name="lanDriverDd">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="displaydriGroupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>251</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>顯示卡</string>
     </property>
     <widget class="QComboBox" name="displayDriverDd">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="otherdriGroupBox">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>10</y>
       <width>201</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>其他</string>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>181</width>
        <height>101</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="otherdriLayout"/>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>450</y>
      <width>181</width>
      <height>86</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="actionsLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="actionCbLayout">
       <item>
        <widget class="QCheckBox" name="atInstallCb">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::DefaultContextMenu</enum>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>自動安裝</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="atShutdownCb">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="statusTip">
          <string>成功安裝所有軀動程式後自動關機</string>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>自動關機</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QVBoxLayout" name="actionBtnLayout">
       <item>
        <widget class="QPushButton" name="diskMgtBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>開啟磁碟管理員</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="installBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>微軟正黑體</family>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="focusPolicy">
          <enum>Qt::WheelFocus</enum>
         </property>
         <property name="statusTip">
          <string>安裝已選取的軀動程式</string>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>執行</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="statusTb">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>410</y>
      <width>201</width>
      <height>181</height>
     </rect>
    </property>
    <property name="statusTip">
     <string notr="true">運行狀態顯示區</string>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="accessibleDescription">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>461</width>
      <height>231</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QScrollArea" name="hwInfoBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>461</width>
       <height>201</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustIgnored</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>442</width>
        <height>199</height>
       </rect>
      </property>
      <property name="statusTip">
       <string>硬件配置顯示區</string>
      </property>
      <widget class="QTextBrowser" name="hwInfoTextBox">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>441</width>
         <height>201</height>
        </rect>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="statusTip">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QLabel" name="hwInfoLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微軟正黑體</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>電腦硬件配置</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>0</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="statusTip">
      <string>重新載入硬件配置</string>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>pic/refresh.png</normaloff>pic/refresh.png</iconset>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>管理</string>
    </property>
    <addaction name="action"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>幫助</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action123">
   <property name="text">
    <string>編輯軀動程式</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>編輯軀動程式</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
